evmrpc/filter.go

```diff
@@ -345,6 +346,11 @@ func (f *LogFetcher) GetLogsByFilters(ctx context.Context, crit filters.FilterCr
 	// Worker function
 	worker := func() {
 		defer wg.Done()
+		defer func() {
+			if r := recover(); r != nil {
+				err = fmt.Errorf("panic in GetLogsByFilters worker: %v", r)
+			}
```

Comment:
same here. Right now if this worker looks at a bunch of heights and the logs on one of those heights is not retrievable, the only thing returned out be the err for that one height. Can we write this in a way such that the worker will do a best effort and return all the real results + errors for any panic-ed responses? Also would be great if we included the exact height on the error response.

Reply:
Fixed